types/maxmsp/index.d.ts

```diff
@@ -1219,6 +1223,69 @@ declare class Maxobj {
      * others.
      */
     understands(message: string): boolean;
+
+	/**
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/maxmsp/index.d.ts;
  Line: 1227;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.